server_addon/nuke/server/settings/publish_plugins.py

```diff
@@ -166,6 +166,18 @@ class BakingStreamModel(BaseSettingsModel):
 
 
 class ExtractReviewDataMovModel(BaseSettingsModel):
+    """[deprecated] use Extract Review Data Baking
+    Streams instead.
+    """
+    enabled: bool = Field(title="Enabled")
+    viewer_lut_raw: bool = Field(title="Viewer lut raw")
+    outputs: list[BakingStreamModel] = Field(
+        default_factory=list,
+        title="Baking streams"
+    )
+
+
+class ExtractReviewIntermediatesModel(BaseSettingsModel):
     enabled: bool = Field(title="Enabled")
     viewer_lut_raw: bool = Field(title="Viewer lut raw")
     outputs: list[BakingStreamModel] = Field(
```

Comment:
I believe since we had renamed it to **Intermediates** any trace of **BakingStream** can be renamed too, right? In this case `IntermediateOutputModel`.